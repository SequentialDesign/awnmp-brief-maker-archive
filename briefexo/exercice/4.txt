The previous example made lots of briefs, in a way that allows to add new ones. BUT, what if you want to add verbs like "work"?

The problem is that it'd work well for I/you/they but will lack a 's' for she/he.

Modify the previous pre-dictonary (ex 3) to add:
- "mk" for "make", "d" for "do", "lv" for "love"
