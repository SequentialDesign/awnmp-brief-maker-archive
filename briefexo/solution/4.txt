(i: I|y: you| {,third,,}(h: he|s: she) |t: they)
(| " " (m: must | ho: should | ha: shall | c: can | co: could | ((lv: love| mk: make| (d: do) {third,,,}("e")) {third,,,}("s"))))
