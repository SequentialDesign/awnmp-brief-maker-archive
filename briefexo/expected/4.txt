::i::I
::y::you
::h::he
::s::she
::t::they
::im::I must
::ym::you must
::hm::he must
::sm::she must
::tm::they must
::iho::I should
::yho::you should
::hho::he should
::sho::she should
::tho::they should
::iha::I shall
::yha::you shall
::hha::he shall
::sha::she shall
::tha::they shall
::ic::I can
::yc::you can
::hc::he can
::sc::she can
::tc::they can
::ico::I could
::yco::you could
::hco::he could
::sco::she could
::tco::they could
::ilv::I love
::ylv::you love
::tlv::they love
::imk::I make
::ymk::you make
::tmk::they make
::id::I do
::yd::you do
::td::they do
::hlv::he loves
::slv::she loves
::hmk::he makes
::smk::she makes
::hd::he does
::sd::she does
